src/Controls/tests/Core.UnitTests/BindingUnitTests.cs

```diff
@@ -1851,6 +1851,8 @@ public void SetBindingContextAfterInnerBindingsAndContextBinding()
 		[Test]
 		public void Convert()
 		{
+			System.Threading.Thread.CurrentThread.CurrentCulture = System.Threading.Thread.CurrentThread.CurrentUICulture = CultureInfo.InvariantCulture;
```

Comment:
Hold on - is this a bug in the test, or a bug in the product? I think there are many places in Xamarin.Forms and MAUI where the product code is using CurrentCulture to parse/format strings, where it should be using Invariant/Ordinal. Not sure if that's the case here.

Reply:
no need to hold on here. it's an issue in the test. The next test in that file is named `ConvertIsCultureAware` and is testing this, precisely